contracts/examples/SampleCrowdsale.sol

```diff
@@ -30,13 +31,14 @@ contract SampleCrowdsaleToken is MintableToken {
  * After adding multiple features it's good practice to run integration tests
  * to ensure that subcontracts works together as intended.
  */
-contract SampleCrowdsale is CappedCrowdsale, RefundableCrowdsale {
+contract SampleCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale {
 
-  function SampleCrowdsale(uint256 _startTime, uint256 _endTime, uint256 _rate, uint256 _goal, uint256 _cap, address _wallet, MintableToken _token) public
+  function SampleCrowdsale(uint256 _startTime, uint256 _endTime, uint256 _rate, address _wallet, uint256 _cap, MintableToken _token, uint256 _goal) public
     CappedCrowdsale(_cap)
     FinalizableCrowdsale()
     RefundableCrowdsale(_goal)
-    Crowdsale(_startTime, _endTime, _rate, _wallet, _token)
+    TimedCrowdsale(_startTime, _endTime)
+    Crowdsale(_rate, _wallet, _token)
```

Comment:
I notice that we should probably be following the "most basic class first" rules here, like we do in the other `*Impl.sol` files. Is there a reason `Crowdsale()` is last? or should it be the first, since it's the most basic class?

Reply:
Actually, it's perhaps that this might not actually work, since the `Crowdsale` is inherited from last and doesn't call `super.*`